### PR TITLE
Interactions 1

### PR DESCRIPTION
**Pull request by Henrywis**
Used the Feedback component to gather likes, dislikes and preferred posts (where preferred posts are posts where a user Starts an Application). This information will be used to create a profile for the user interactions to be used as inputs for the ranking model, along with job listing features.

<div style="position: relative; padding-bottom: 56.25%; height: 0;"><iframe src="https://www.loom.com/embed/294fe3fa93e84d368a33abe643619026?sid=416096b7-04f1-4d6b-b865-3963bec99a05" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>

### CODE REVIEW

File: iRecruit-ui/src/components/Main/Main.jsx [hunk 1]
```diff
@@ -109,6 +114,27 @@ function Main() {
...
+
+  // Function to handle clicking on "Like" button
+  const handleLike = (job) => {
+    setLikedPosts((prevLikedPosts) => [...prevLikedPosts, job]);
+  };
+
+  // Function to handle clicking on "Dislike" button
+  const handleDislike = (job) => {
+    setDislikedPosts((prevDislikedPosts) => [...prevDislikedPosts, job]);
+  };
```
Comment (9600baud) [inline iRecruit-ui/src/components/Main/Main.jsx:136]: Look up the `useCallback` hook if you want some more advanced React topics :)

File: iRecruit-ui/src/components/Feedback/Feedback.jsx [hunk 2]
```diff
@@ -1,11 +1,55 @@
 import React from "react";
 import "./Feedback.css";
```
Comment (9600baud) [inline iRecruit-ui/src/components/Feedback/Feedback.jsx:2]: You dont necessarily need to split CSS from the component.  Its debatable whether one is preferred over the other but just letting you know :)

**Review by 9600baud — COMMENTED** [review]
keep it up

File: iRecruit-ui/src/components/Main/Main.jsx (hunk 1 above)
Comment (Henrywis) [reply iRecruit-ui/src/components/Main/Main.jsx:136]: Okay, sure! Thank you

File: iRecruit-ui/src/components/Feedback/Feedback.jsx (hunk 2 above)
Comment (Henrywis) [reply iRecruit-ui/src/components/Feedback/Feedback.jsx:2]: Do you mean I could have the styles incorporated with the component itself? Like style them as I create in the JSX file?

**Pull request merged**